app/models/whitehall_migration/timeline_entry.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class WhitehallMigration::TimelineEntry < ApplicationRecord
```

Comment:
I think this class would make more semantic sense to live under the TimelineEntry namespace as despite being a product of a WhitehallMigration it has very little actual association with it and is likely to exist forever. Whereas at some point we'd hope we could kill off all the WhitehallMigration classes.

I'd think something like TimelineEntry::WhitehallImportedEntry or TimelineEntry::WhitehallEntry would be a good option.

Reply:
Good point about `WhitehallMigration` eventually going away, so I've decided on `TimelineEntry::WhitehallImportedEntry` (having gone through many iterations of different names).